tests/cypress/e2e/jahiaUserLanguage1.cy.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    abortAllWorkflows,
+    createSite,
+    createUser,
+    deleteSite,
+    deleteUser,
+    grantRoles,
+    publishAndWaitJobEnding,
+    setNodeProperty,
+    startWorkflow,
+    unpublishNode,
+} from '@jahia/cypress'
+import { PageComposer } from '../page-object/pageComposer'
+
+const siteKey = 'jahiaUserLanguageSite'
+const editorLogin = 'jahiaUserLanguageEditor'
+const editorPassword = 'password'
+const publisherLogin = 'jahiaUserLanguagePublisher'
+const publisherPassword = 'password'
+
+const expect404 = (url: string) => {
+    cy.request({
+        url: url,
+        failOnStatusCode: false,
+    }).then((response) => {
+        expect(response.status).to.eq(404)
+    })
+}
+
+const expect404InAllLanguages = () => {
+    expect404(`/en/sites/${siteKey}/home/pageen.html`)
+    expect404(`/fr/sites/${siteKey}/home/pageen.html`)
+    expect404(`/es/sites/${siteKey}/home/pageen.html`)
+}
+
+describe('Jahia user language 1 testsuite', () => {
```

Comment:
We would need a more meaningful description for both the test suite and the test case.